test/Libraries/Microsoft.Extensions.AI.Abstractions.Tests/Contents/FunctionCallContentTests..cs

```diff
@@ -84,48 +81,6 @@ public void Constructor_PropsRoundtrip()
         Assert.Same(e, c.Exception);
     }
 
-    [Fact]
```

Comment:
Should this be replaced with a test checking that the roundtripped value returns null? Or do we not need to specify this?

Reply:
Sure